lib/msal-common/src/client/SilentFlowClient.ts

```diff
@@ -43,13 +41,12 @@ export class SilentFlowClient extends BaseClient {
         request: CommonSilentFlowRequest
     ): Promise<AuthenticationResult> {
         try {
-            const cachedToken = await this.acquireCachedToken(request);
+            const [authResponse, cacheOutcome] = await this.acquireCachedToken(
+                request
+            );
```

Comment:
```suggestion
            const [authResponse, cacheOutcome] = await this.acquireCachedToken(request);
```

Reply:
I agree with you, but we have some crazy linting rules now that auto-fix the code to look like this.